tests/sql_test_base.py

```diff
@@ -122,52 +66,60 @@ def get_query(self, filter_attribute, filter_value, query=None):
         """
 
         if query is None:
-            query = self.session.query(self.TableClass)
+            query = session.query(self.TableClass)
 
         fa = getattr(self.TableClass, filter_attribute)
         q = query.filter(fa == filter_value).order_by(asc(fa))
         return q
 
-    def test_count(self, data_name, expected_count):
+    def check_count(self, data_name):
         """
         Test the record count of a data type
         """
-        q = self.get_query(self.count_attribute, data_name)
-        records = q.all()
-        assert len(records) == expected_count
-
-    def test_value(self, data_name, attribute_to_check, filter_attribute, filter_value, expected):
+        with db_session_with_credentials(
+                self.database_name(), self.CREDENTIAL_FILE
+        ) as (session, engine):
+            q = self.filter_measurement_type(session, data_name)
+            records = q.all()
+        return len(records)
+
+    def check_value(
```

Comment:
Just to be clear, these wont be used as tests without the keyword test in the name.